.gitignore

```diff
@@ -10,3 +10,4 @@ contract
 !.yarn/sdks
 !.yarn/versions
 dist
+ethereum_history_api/.DS_Store
```

Comment:
This is macOS specific file, I propose to add this to `~/.gitignore_global`

Reply:
OK, I agree to leave it as we all work on macOS

Reply:
Just
```
.DS_Store
```
(will work for subfolders)